yt_dlp/extractor/common.py

```diff
@@ -1503,6 +1503,7 @@ def extract_video_object(e):
             author = e.get('author')
             info.update({
                 'url': url_or_none(e.get('contentUrl')),
+                'embedUrl': url_or_none(e.get('embedUrl')),
```

Comment:
info must obey extractor specifications. `embedURL` is not a defined field

Reply:
updated this. now injects `embedUrl` into the `formats` field instead.